Part5 (#62)
FuelCycleFIT/FITProject: PAPER/FIT_Paper_Ex1/Part_5.tex

\section{Conclusions and perspectives}

Substantial resources have been invested in a variety of Nuclear Fuel Cycle
tools, whether to serve specific purposes, or as they expanded to more general
purpose, but confidence in each new analysis remains fragile because of a lack
of generic ability to develop confidence in such tools.
Fuel cycle simulators can be used to answer different questions that requires
different level of precision. To increase the confidence level of studies, all
institutions are willing to increase the level of complexity of the software
even if it does not necessary improve the precision of the calculations. 

This paper presents the FIT project which aims to provide a rigorous scientific
tools allowing researchers/developers/analysts to determine whether or not a
feature, a functionality, a model refinement is required to perform a nuclear
fuel cycle study within a certain precision goal.

The FIT project approach consists in testing code functionality by an intra-code
comparison of calculation with and without the feature. The FIT project is not a

> benchmark and there is inter-simulator comparison but aims to perform

Reviewer: ```suggestion
benchmark and there is no inter-simulator comparison but aims to perform
```

> intra-simulator comparison, comparing the simulator output obtain with a single

Author: ```suggestion
intra-simulator comparison, comparing the simulator output obtained with a single
```

simulator, with and without the feature/functionality to test. The strength of
this project relies on the diversity of simulator used to solve the different
problems allowing to reach nearly simulator-agnostic conclusion. Following this
principle, the FIT project will provide informations about which functionalities
are required according to the question that needs to be answered and the
precision level to reach.

The work presented here is dedicated fuel cycle simulators ability to vary fresh
fuel compositions according to the available materials. Two treatments have been
considered: a fixed enrichment for fresh fuel, regardless of the isotopic

> composition, and an model use to adjust the fresh fuel composition in accordance

Reviewer: ```suggestion
composition, and the use of a model to adjust the fresh fuel composition in accordance
```

to the isotopic composition. In this first exercise, the output estimators focus
on availability of the plutonium in the fuel cycle, so will probably be more
valuable for fuel cycle study involving plutonium recycling.

In agreement with the FIT philosophy, all calculations were performed with a
fixed fraction and with the use of a model that calculate fissile enrichment for
fresh fuels. Inter-software comparison have been done but each code is tested

> against itself, \gls{FF} vs \gls{FLM} on 2 different reactor, across 5 different

Reviewer 2: reactorS ?

> against itself, \gls{FF} vs \gls{FLM} on 2 different reactor, across 5 different

Author: ```suggestion
against itself, \gls{FF} vs \gls{FLM} on 2 different reactors, across 5 different
```

simulators, (CLASS, ANICCA, TR\_EVOL and CYCLUS) for the \gls{PWR} and (CLASS,
TR\_EVOL, JOSETTE) for the \gls{SFR}. The importance of adjusting fresh fuel
compositions have been tested around 3 main questions : 
\begin{itemize}

>     \item the global plutonium inventory estimation,

Reviewer 2: Not sure to agree with those three item. For me, item 1 and 2 are quite similar. And for the item 3, why focus on spent fuel...? Since all the spent fuel front end will be impacted.
